packages/api-clients/open-api/tenantApi.yml

```diff
@@ -808,6 +811,9 @@ paths:
               $ref: "#/components/schemas/CertifiedTenantAttributeSeed"
         required: true
       responses:
+        headers:
+          X-Metadata-Version:
+            $ref: "#/components/headers/MetadataVersionHeader"
```

Comment:
Missing the header declaration in components (see https://github.com/pagopa/interop-be-monorepo/pull/1848/files#diff-a50f62f3a4939e0ad8d4f443bf76522090263ad0bfabf14e174d8d83475fc372R406)